mon-pix/translations/en.json

```diff
@@ -125,6 +125,12 @@
         "label": "Search for an organization",
         "placeholder": "Select an organization",
         "searchLabel": "Keyword search"
+      },
+      "other-authentication-providers": {
+        "signup-heading": "Other means of registration",
```

Comment:
La traduction présente dans le ticket est `Other methods of registrations`

Reply:
Suivant les dernières instructions, c'est la formulation _« Other ways to sign up »_ qui est utilisée en attendant de revoir toutes les formulations en anglais.